docs/configuration/ScenarioDeploymentConfiguration.md

```diff
@@ -316,21 +316,20 @@ Deployment Manager of type `lite-embedded` has the following configuration optio
+| shouldCheckAvailableSlots                   | boolean        | true          | When set to true, Nussknacker checks if there are free slots to run new job. This check should be disabled on Flink Kubernetes Native deployments, where Taskmanager is started on demand.                                                  |
+| waitForDuringDeployFinish.enabled           | boolean        | true          | When set to true, after Flink job execution, we check if tasks were successfully started on TaskMangers, before marking version as deployed. Otherwise version is marked as deployed immediately after successful response from JobManager. |
+| waitForDuringDeployFinish.maxChecks         | boolean        | 180           | It works when `waitForDuringDeployFinish.enabled` option is set to `true`. This parameter describe how many times we should check if tasks were successfully started on TaskMangers before notifying about deployment failure.              |
+| waitForDuringDeployFinish.delay             | boolean        | 1 second      | It works when `waitForDuringDeployFinish.enabled` option is set to `true`. This parameter describe how long should be delay between checks.                                                                                                 |
+| scenarioStateCaching.enabled                | boolean        | true          | Enables scenario state caching in scenario list view                                                                                                                                                                                        |
+| scenarioStateCaching.cacheTTL               | duration       | 10 seconds    | TimeToLeave for scenario state cache entries                                                                                                                                                                                                |
+| scenarioStateRequestTimeout                 | duration       | 3 seconds     | Request timeout for fetching scenario state from Flink                                                                                                                                                                                      |
+| jobConfigsCacheSize                         | int            | 1000          | Maximum number of cached job configuration elements.                                                                                                                                                                                        |
+| miniCluster.useForScenarioTesting           | boolean        | true          | Creates mini cluster once and reuses it for each scenario testing attempt                                                                                                                                                                   |
```

Comment:
imo previous `reuseMiniClusterForScenarioTesting` and `reuseMiniClusterForScenarioStateVerification` names were longer but more meaningful for both `true` and `false` states. Now when u set `miniCluster.useForScenarioTesting=false` NU still uses miniCluster for ScenarioTesting but it is not shared.

Reply:
You've right, reverted